### PR TITLE
Catch IOError in load_notebook and add logging to the python side...

### PR DESCRIPTION
**Pull request by damianavila**
Prompted by https://github.com/bokeh/bokeh/pull/1742#issuecomment-71072802, I needed to add some logging capabilities in the python side.

@mattpap @bryevdv

### CODE REVIEW

**Comment by mattpap** [comment]
I had a lazy load approach in mind (not sure if actually possible), but that's fine for now. At least I can import bokeh this way.

**Comment by damianavila** [comment]
OK, housekeeping, I also removed the PYTHONLIB env variable...

I would like to merge this one today unless someone has other thoughts.

File: bokeh/settings.py [hunk 1]
```diff
@@ -43,12 +44,19 @@ def minified(self, default=None):
     def log_level(self, default=None):
         return self._get_str("LOG_LEVEL", default)
 
+    def py_log_level(self, default='info'):
+        level = self._get_str("PY_LOG_LEVEL", default)
+        LEVELS = {'trace': logging.NOTSET,
```
Comment (bryevdv) [inline bokeh/settings.py]: what does this even mean? What does `logging.NOTSET` do? For sure `trace` is  meaningful for the JS log library we use but I think maybe this should be removed as an option on the python side.

Comment (damianavila) [inline bokeh/settings.py]: > What does logging.NOTSET do? 

https://docs.python.org/3.4/library/logging.html#logging.Logger.setLevel

> For sure trace is meaningful for the JS log library we use but I think maybe this should be removed as an option on the python side.

I could be meaningful too from the Python side, but I agree is probably one of the less used levels... I can remove it if you feel is convenient...

Comment (bryevdv) [inline bokeh/settings.py]: Well reading that it seems complicated and confusing, so I'd say remove it just on those grounds.

Comment (damianavila) [inline bokeh/settings.py]: Done!

**Comment by bryevdv** [comment]
LGTM, merging.

**Pull request merged**